holidayextras/ui-toolkit
# PATCH adding a role to the countdown and fixing padding issues

#### What does this PR do? (please provide any background)
- Removes the padding from the countdown timer labels and adds a space (react creates an extra span, we will just have to suck it up)

#### What tests does this PR have?
Added a test to check for `role=timer`

#### How can this be tested?
- Run `grunt docs` and go down to the countdown component on the UI Toolkit
- Right click on the element and check it has `role=timer`

#### Screenshots / Screencast
none

#### What gif best describes how you feel about this work?
![dog](https://media.giphy.com/media/TmVqs1EEKnFjq/giphy.gif)

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
